charts/brigade-project/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: Create a Brigade project
 name: brigade-project
-version: 0.18.0
+version: 0.18.1
```

Comment:
It looks like this value is used for sidecar image tags, which don't exist, naturally
```
  Normal   BackOff    13s   kubelet, aks-nodepool1-26912603-0  Back-off pulling image "deis/git-sidecar:v0.18.1"
  Warning  Failed     13s   kubelet, aks-nodepool1-26912603-0  Error: ImagePullBackOff
```
I say we bump this back to `0.18.0` here and in the `brigade/Chart.yaml` and we'll just bump to the next tagged version once this changeset is included in a release.

Reply:
I've added the `AppVersion` to be used for the image tag, in order to allow the chart to evolve.

This is the way it's done on the brigade chart, also It's what I understand is the best way to do it, after reading the helm documentation: https://docs.helm.sh/developing_charts/#charts-and-versioning and https://docs.helm.sh/developing_charts/#the-appversion-field